### PR TITLE
Final changes for glvis-4.0

### PR DESCRIPTION
**Pull request by tzanio**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -8,53 +8,61 @@
...
+  with planar faces.
+
+- Updated to support the display and slicing of meshes with wedge elements.
+
+- Improved the opening of parallel meshes/solutions.
+
+- Transparency and printing in textured coloring modes is now supported, and
+  no-texture coloring has been removed as a result.
+
+- Text on screen is now rendered using a texture atlas.
```
Comment (kanye-quest) [inline CHANGELOG:53]: We could probably remove this point, unless it's to mention that freetype is now a required dependency

Comment (tzanio) [reply CHANGELOG:53]: How about

```
- FreeType is now a required dependency and text on screen is rendered using a
  texture atlas.
```

Comment (kanye-quest) [reply CHANGELOG:53]: That sounds good to me

**Review by rw-anderson — APPROVED** [review]

**Comment by tzanio** [comment]
@kanye-quest,  @tomstitt, @v-dobrev  and @acfisher -- please review and test.

**Review by kanye-quest — APPROVED** [review]
Tested as working with Windows (via WSL) + VcXsrv

**Review by tomstitt — APPROVED** [review]
I renamed the `glvis-jupyter` repository to `glvisjs` and removed Jupyter related work from that repo. The build is also fixed and now works in one pass. *glvis.js* was updated with the latest master of MFEM and GLVis.

File: CHANGELOG [hunk 2]
```diff
@@ -8,53 +8,62 @@
...
-
-- Transparency and printing in textured coloring modes is now supported, and
-  no-texture coloring has been removed as a result.
-
-- Text on screen is now rendered using a texture atlas.
+  "-oldgl" can be used to request the legacy backend.
 
 - Preliminary support for building GLVis to Javascript/WebAssembly using
-  Emscripten.
+  Emscripten, see https://github.com/GLVis/glvis-jupyter.
```
[the remark this answers is not in the document]
Comment (tzanio) [reply CHANGELOG]: I have a slight preference for `glvis-js` but `glvisjs` is fine too.

[the remark this answers is not in the document]
Comment (tzanio) [reply CHANGELOG]: We decided to go with `glvis-js`, we can update that later if necessary

**Review by v-dobrev — APPROVED** [review]

**Pull request merged**